reference/configuration.html.md.erb

```diff
@@ -618,3 +618,7 @@ This overrides the `CMD` set by the Dockerfile. It should be specified as an arr
 ### `entrypoint`
 
 This overrides the `ENTRYPOINT` set by the Dockerfile. It should be specified as an array of strings, as seen in the example above.
+
+### `exec`
+
+This overrides the `EXEC` set by the Dockerfile. It should be specified as an array of strings, as seen in the example above.
```

Comment:
There is no "EXEC" statement in Dockerfile. 

If set, the `exec` fly.toml directive overrides both, the ENTRYPOINT and CMD.